src/lib/Repository/Validator/TargetContentValidator.php

```diff
@@ -51,13 +45,11 @@ public function validate(int $value, array $allowedContentTypes = []): ?Validati
                     'targetContentId'
                 );
             }
-        } catch (NotFoundException | UnauthorizedException $e) {
+        } catch (NotFoundException $e) {
             return new ValidationError(
-                'Content with identifier %contentId% is not a valid relation target',
```

Comment:
What is the reason behind this change ?

Reply:
That was @glye idea, please see the comments on the JIRA ticket.